CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
-## v0.1.0 (Released 2021-08-22)
+## v0.0.2 (Release 2023-07-24)
+
```

Comment:
Can we release v0.1.0 ?

Reply:
good call, updated https://github.com/moov-io/fips-state-codes/pull/8/commits/d353b57cbbd95411cb0aea2c0b6891d05ca625b7